### ISSUE TITLE
FCN trained on cityscapes

### ISSUE THREAD

**Issue by vigorous0503** [mcordts/cityscapesScripts#12]
Hi Mcordts,

Because I want to make a test on the FCN and I find you have trained the FCN on cityscapes in the page of the official benchmark suite ,  I want to ask you whether the model still keep.
If yes, can you release the model? 

Thank you!

**Comment by mcordts** [comment]
Hi,

unfortunately, I cannot release the model.

Best,
Marius

2016-10-15 11:53 GMT+02:00 B.C Tsai <notifications@github.com>:

**Comment by nitahhhh** [comment]
Hi Mcordts,

Did you train the FCN on cityscapes-dataset with balanced weighting for classes?
Thanks.

**Comment by mcordts** [comment]
Hi,
I did not use weight balancing.
Best,
Marius

2016-11-01 6:42 GMT+01:00 nitahhhh <notifications@github.com>:

**Comment by nitahhhh** [comment]
Hi Mcordts,
I have tried to train by FCN but the performance was not as good as yours. 
Can you share some parameters  such as learning rate.
Thanks.

BR,

**Comment by mcordts** [comment]
Hi,

did you check the parameters and training details that are in the paper,
especially in the supplementary material?

The learning rate was 2.38e-11 with an unnormalized cross entropy loss on
images with size 1024x1024. Note that we split the images into two halves
of size 1024x1234 each and ignored the overlap of 210 pixels in the loss.

Best,
Marius

2016-11-01 15:52 GMT+01:00 nitahhhh <notifications@github.com>:

**Comment by nitahhhh** [comment]
Hi Mcordts,

Cool.
Thank you very much！

BR.

**Comment by codyreading** [comment]
Hi Mcordts,

When training using FCN on cityscapes, was the same learning rate used for the experiment when the images were downscaled by a factor of 2? 

Additionally, have you considered training Cityscapes using the "at-once" FCN-8s architecture? I believe this was not released at the time of the paper, but was just wondering your thoughts.

Best regards,
Cody Reading
